source/manual/deploying-terraform.html.md

```diff
@@ -9,77 +9,78 @@ review_in: 3 months
+- `*-administrators` can deploy anything.
 
-You can find which class of user you are (that is, which roles your ARN has been assigned to) in the
-[infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
+You can find which class of user you are [infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
 
-### 2. Install `gds-cli`
+## Deploying on Jenkins
 
-See [instructions for installing `gds-cli`](/manual/access-aws-console.html).
+> Note that this requires you to push your changes to GitHub.
 
-### 3. Create a read-only GitHub personal access token
+### 1. Generate an AWS token and key
 
-Create a [Github personal access token](https://github.com/settings/tokens) with the `read:org` scope.
+First make sure you have [`gds-cli`](/manual/access-aws-console.html) installed.
 
-  * Follow the link above and press the `Generate a new token` button near the
-    top of the page.
-  * Enter a suitable description for the token under `Note`, for example `Token
-    for Jenkins Terraform deployments`.
-  * Under `Select scopes`, tick the `read:org` box **only**.
-  * Press `Generate token` at the bottom of the form.
-  * Please take care to store and handle the token securely. It allows access
-    to Jenkins. If you accidentally share your token,
-    [revoke it immediately](https://github.com/settings/tokens) and follow the
-    [instructions for reporting a potential data security incident][security-incidents].
+👉 [Instructions for installing `gds-cli`](/manual/access-aws-console.html)
 
-[security-incidents]: https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/working-at-the-white-chapel-building/security/security-incidents
+Run the following to generate a token and key.
 
-## Plan and deploy a Terraform project via Jenkins using `deploy.rb`
+```
+gds aws <your-role e.g. govuk-integration-poweruser> -e
+```
 
-The Ruby script [`tools/deploy.rb`][deploy-rb] in the `govuk-aws` repository takes care of requesting temporary AWS credentials with an assumed role and queuing the deployment Jenkins job.
+### 2. Run the Jenkins job
 
-To follow these instructions you will need:
+**Always `plan` first, check that the output is what you expect, then `apply`.**
 
-* The name of the Terraform project which you want to deploy, for example `app-backend`.
-* The stackname of the Terraform project which you want to deploy. Usually, this is `blue` for `app-` projects and `govuk` for `infra-` projects.
-* Your GitHub username.
-* Your GitHub personal access token (see setup instruction above).
-* The name of your AWS role (`govuk-powerusers` or `govuk-adminstrators`).
-* A local copy of the `govuk-aws` Git repo, which contains the `deploy.rb` script. The example command assumes that your local repo is in the directory `~/govuk/govuk-aws`.
+Manually run the Jenkins job, pasting the token and key from the previous step.
```

Comment:
We shouldn't be recommending this way. The "use the deploy script" way to trigger the Jenkins job should be the default that people do. With the `./tools/deploy.rb` script on the end of the gds-cli command, people don't have to have tokens on their clipboards and accidentally paste them into Slack or somesuch.

Reply:
IMO copying and pasting three tokens six times (or even three times) is more effort to find the beginning and end, etc. etc. than adding a GitHub token once.

Reply:
Having now copied and pasted tokens many times, I agree. However, I also found it more intuitive as a starting point, because now I know what the deploy script is doing and that makes me more confident when using it. I've added a caveat into the section that I'm hoping captures the discussion here. Does that address the issue for you?